docker-compose.yaml

```diff
@@ -7,5 +7,11 @@ services:
     working_dir: /go/src/github.com/ismaelchess/go-share-secret
     volumes: 
       - .:/go/src/github.com/ismaelchess/go-share-secret
+    links:
```

Comment:
No need to create links. That's old docker. Name of `service` becomes the `host`, so you can simply connect using that across other services